js/Game.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+class Game extends Drawable {
+  constructor(width, height) {
+    super(); 
+
+    this.width = width;
+    this.height = height;
+
+    this.snake = null;
+    this.fruit = null;
+
+    this.score = null;
+    this.gameOver = false;
+
+    document.addEventListener('keydown', (e) => this.onKeyDown(e));
+  }
+
+  init() {
+    this.snake = this.createSnake();
+    this.fruit = new Fruit(this.getRandomFruitPosition());
+
+    this.score = 0;
+    this.gameOver = false;
+  }
+
+  createSnake() {
+    const x = rand(3, 17);
+    const y = rand(3, 17);
+    
+    const snakePoint = new Point(x, y);
+
+    return new Snake(snakePoint, 3);
+  }
+
+  getRandomFruitPosition() {
+    let fruitPosition = getRandomPoint(1, 20);
+    const isSnake = this.snake.getCells();
+
+    for (let i = 0; i < isSnake.length - 1; i++) {
+      if (fruitPosition.eql(isSnake[i])) {
+        fruitPosition = getRandomPoint(1, 20);
+      }  
+    }
+
+    return new Point(fruitPosition.x, fruitPosition.y);
```

Comment:
What if `getRandomPoint` returns twice the same?

Reply:
Let's go for a recursive function then :)